lib/platform/referralTransaction.js

```diff
@@ -48,7 +59,7 @@ const schema = schemaCreator.createSchema(
      * @name notes
      * @desc If something goes wrong, the notes contains the reason.
      * @type String
-     * @memberof Referrals
+     * @memberof ReferralTransaction
      * @inner
      */
     notes: {
```

Comment:
This can be errorMessage or something similar, but let's talk more about the architecture tomorrow.